validator/sawtooth_validator/execution/context_manager.py

```diff
@@ -320,6 +320,8 @@ def __init__(self, database, state_delta_store):
 
         self._address_regex = re.compile('^[0-9a-f]{70}$')
 
+        self._namespace_regex = re.compile('^[0-9a-f]{0,70}$')
```

Comment:
With respect to 'Each byte is an address path element in the radix addressing scheme". That is how it is currently, but I thought that STL-149 would be changing that, potentially. Or would it be that at minimum the address path element would be 1 byte and potentially more.

I'm trying to think of a situation where it might make sense for the input or output to reference a non-subtree. It would be odd, because a client couldn't query it. If it make sense to you to stick with the even number of characters in the namespace requirement I'll change it.